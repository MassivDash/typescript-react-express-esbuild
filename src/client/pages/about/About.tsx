import React from 'react';


const About: React.FunctionComponent = () => {
    return <div>About</div>;
};

export default About;
