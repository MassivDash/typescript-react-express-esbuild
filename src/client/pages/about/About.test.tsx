import { render } from '@testing-library/react';
import React from 'react';
import About from './About';

describe('About', () => {
    
    it('should render', () => {
        const { asFragment, queryByText } = render(<About />);

        expect(asFragment()).toMatchSnapshot();
        expect(queryByText('About')).toBeTruthy();
    });
});
