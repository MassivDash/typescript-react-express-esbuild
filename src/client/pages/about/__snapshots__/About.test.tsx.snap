// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render 1`] = `
<DocumentFragment>
  <main
    class="mt-10 mx-auto max-w-7xl px-4 sm:mt-12 sm:px-6 md:mt-16 lg:mt-20 lg:px-8 xl:mt-28"
  >
    <div>
      About
    </div>
  </main>
</DocumentFragment>
`;
